subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -553,6 +554,9 @@ static void isr_rx(void *param)
 		/* BIS index */
 		bis_idx = lll->bis_curr - 1U;
 
+		/* Current stream */
+		stream_curr = lll->stream_curr;
+
```

Comment:
`stream_curr` is never used before it gets overridden in line 595

Reply:
There is a goto just below here, when the reception is done without a PDU being received (PDU missed for the current stream).

Reply:
Ah, missed that. Damn `goto`s :D

Reply:
github diff can hide context...

Regarding use of `goto` needs some thought... hopefully I have used it for common exit paths and not complicated readability.

Nice read here: https://koblents.com/Ches/Links/Month-Mar-2013/20-Using-Goto-in-Linux-Kernel-Code/

Reply:
Yeah, it was probably a mix of `goto`, Github diff and the size of the function (in lines of code) :)